Makefile

```
@@ -9,14 +9,15 @@ kernel.bin: $(kernel_src)
 apps/%.bin: apps/%.c
 	make -C apps $(patsubst apps/%, %, $@)
 
-initrd: tools/create_initrd.py $(apps) libc/build/libc.so libmath/build/libmath.so
+initrd: tools/create_initrd.py $(apps) libc/build/libc.so libmath/build/libmath.so libgraphics/build/libgraphics.so
```

checkmake (0.3.2)

[warning] Target body for "initrd" exceeds allowed length of 5 lines (11).

(maxbodylength)